### PR TITLE
fix PAL connections count

### PR DESCRIPTION
**Pull request by howard-stearns**
expose totalPages and totalEntries, and use the later to display connections count

Fixes https://highfidelity.fogbugz.com/f/cases/16528/

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11224/HighFidelity-Beta-PR13575-886df95.apk

File: interface/resources/qml/hifi/models/PSFListModel.qml [hunk 1]
```diff
@@ -79,8 +83,10 @@ ListModel {
         if (response.current_page && response.current_page !== currentPageToRetrieve) { // Not all endpoints specify this property.
             return fail("Mismatched page, expected:" + currentPageToRetrieve);
         }
+        totalPages = response.total_pages;
+        totalEntries = response.total_entries;
```
Comment (zfox23) [inline interface/resources/qml/hifi/models/PSFListModel.qml]: Are these two entries guaranteed to be present in the response? Would it be safer to do `totalEntries = response.total_entries || 0` etc ?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11224/HighFidelity-Beta-PR13575-886df95.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11224/HighFidelity-Beta-Interface-PR13575-886df95.dmg

File: interface/resources/qml/hifi/models/PSFListModel.qml (hunk 1 above)
Comment (howard-stearns) [reply interface/resources/qml/hifi/models/PSFListModel.qml]: good catch. QML doesn't behave like C when storing 'undefined' in an int, nor like Javascript in a var. It complains noisily.  I'll change it.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11225/HighFidelity-Beta-PR13575-8b713a4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11225/HighFidelity-Beta-Interface-PR13575-8b713a4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11225/HighFidelity-Beta-PR13575-8b713a4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11225/HighFidelity-Beta-Interface-PR13575-8b713a4.exe

**Comment by howard-stearns** [comment]
developer tested

**Pull request merged**